src/docs/santiment-queries/exploration/index.md

```diff
@@ -149,14 +149,14 @@ The complete table name is `default.intraday_metrics`, where `default` refers to
 
 The columns are represented by their names and corresponding data types. Data types may have modifiers that determine how the data is stored on disk (e.g., `CODEC(DoubleDelta, LZ4)`). These modifiers can be disregarded when examining a table.
 
+### ORDER BY
+
+This is a crucial field to understand and consider when writing a query. ClickHouse is a column-oriented database, meaning that the data for columns is stored continuously on-disk. This makes it more challenging and less efficient to use multiple indexes, so the column order in `ORDER BY` is essential for writing performant queries. If filters for the columns at the beginning of the list are present, the query will run faster.
+
```

Comment:
Maybe this can be extended a little bit in order to not be confused with the `ORDER BY` when writing SQL queries.

```suggestion
### ORDER BY

This should not be confused with `ORDER BY` clause used when you write an SQL query, which controls how the records in the result set are sorted. The `ORDER BY` examined here is a property of the database table and controls how the data is sorted on disk, acting like an index.

This is a crucial field to understand and consider when writing a query. ClickHouse is a column-oriented database, meaning that the data for columns is stored continuously on disk. This makes it more challenging and less efficient to use multiple indexes, so the column order in `ORDER BY` is essential for writing performant queries. As a rule of thumb, If filters for the columns located at the beginning of the `ORDER BY` list are present, the query will run faster.

```